docs/data/charts/bars/bars.md

```diff
@@ -85,6 +85,21 @@ See [Axis—Grid](/x/react-charts/axis/#grid) documentation for more information
 
 {{"demo": "GridDemo.js"}}
 
+### Color scale
+
+Like other charts, you can modify the [series color](/x/react-charts/styling/#colors) by using series color, or some color palette.
+
+You can also modify color by using axes `colorMap` which maps values to colors.
+Scatter charts use by priority:
```

Comment:
```suggestion
Bar charts use by priority:
```

Or even better to be consistent with the "line chart" page (or remove the "the" there)
 
```suggestion
The bar charts use by priority:
```